action.yaml

```diff
@@ -19,23 +20,25 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - uses: robinraju/release-downloader@v1.3
-      with:
-        repository: "fischerscode/tagger"
-        fileName: "tagger.linux"
-        out-file-path: ${{ github.action_path }}
-        tag: v0.1.0
-    - run: |
-        mv ${{ github.action_path }}/tagger.linux ${{ github.action_path }}/tagger
+    - name: Download tagger
+      run: |
+        if [ "${{ runner.os }}" = "Linux" ]; then
+          suffix=linux
+        elif [ "${{ runner.os }}" = "macOS" ]; then
+          suffix=mac
+        else
+          suffix=windows.exe
+        fi
+        wget https://github.com/fischerscode/tagger/releases/download/v0.1.0/tagger.${suffix} -O ${{ github.action_path }}/tagger
```

Comment:
It seems like wget is not available on windows. https://github.com/fischerscode/action-test/actions/runs/4596897051/jobs/8118932256 (Ignore the failed run for linux and macos.)

Reply:
I just handled "Windows" case in a separate step using `pwsh`. Could you please check? I have doubts regarding `sed` in a last step but let me know if it doesn't work and I will replace `sed` with something from `pwsh` world for "Windows" case.

Reply:
I guess we will see if the `sed` works as expected once the thing [below](https://github.com/fischerscode/tagger/pull/8#discussion_r1177681710) is solved. For now, it looks good to me.